### ISSUE TITLE
Option to implement Django-pipeline with script loader

### ISSUE THREAD

**Issue by commadelimited** [jazzband/django-pipeline#239]
I'm looking into using this codebase in our app to streamline our deployment process. Currently we're using a script loader called [head.js](http://headjs.com/) which helps prevent script blocking, and also offers a basic amount of requirement management. The problem for us would be using `head.js`, whose set up looks like this,:

```
// script loading
head.js(
    // remote files
    "//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js",

   {% compressed_js 'activity' %}

    // head.js callback, executes after all scripts are loaded
    function(){
        // callback function
    }
);
```

in conjunction with django-pipeline which outputs plain script tags. Is there an option to django-pipeline to output it's list of assets as a commadelimited list? Something as simple as that would allow me to mix and match with external libraries take advantage of CDN?

It might look like this:

```
// script loading
head.js(
    // remote files
    "//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js",

    {% compressed_js 'activity' %}

    // head.js callback, executes after all scripts are loaded
    function(){
        // callback function
    }
);
```

which would render like this:

```
// script loading
head.js(
    // remote files
    "//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js",

    // 3rd party libraries
    "js/libs/{{script_path}}/core.libs.js",
    "js/libs/{{script_path}}/moment-2.0.0.js",
    "js/es/{{script_path}}/application.activity.js",

    // head.js callback, executes after all scripts are loaded
    function(){
        // callback function
    }
);
```
